build/ansible/roles/dashboards/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Stop grafana
+  shell: "supervisorctl {{ item }} grafana"
+  become: true
+  become_user: pmm
+  become_method: su
+  loop:
+    - stop
+    - remove
+
 - name: Get plugin list
```

Comment:
```suggestion
- name: List Percona Dashboard plugins
```

Reply:
This will not list them, it only finds the files and saves them to a variable.